src/components/Hub/HubSortDropdown.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import {
+  createStyles,
+  FormControl,
+  InputBase,
+  InputLabel,
+  MenuItem,
+  Select,
+  Theme,
+  withStyles,
+} from "@material-ui/core"
+
+type Props = {
+  selectedSortOption: string
+  sortOptions: string[]
+  handleSortOption: (event: React.ChangeEvent<{ value: string }>) => void
+}
+
+const BootstrapInput = withStyles((theme: Theme) =>
```

Comment:
I think it's better to use emotion to style this too. Also, I'm not sure about the name BootstrapInput.

Reply:
Changed the name to HubSortInput. I couldn't find another way to style the input element of materials InputBase. Maybe let's have another call about it